test/spec/modules/rdnBidAdapter_spec.js

```diff
@@ -0,0 +1,101 @@
+import { expect } from 'chai'
+import * as utils from 'src/utils'
+import { spec } from 'modules/rdnBidAdapter'
+import { newBidder } from 'src/adapters/bidderFactory'
+
+describe('rdnBidAdapter', function() {
+  const adapter = newBidder(spec)
+  const ENDPOINT = 'https://s-bid.rmp.rakuten.co.jp/h'
+
+  describe('inherited functions', () => {
```

Comment:
Could you update the various `, () => {` arrow references used in this file to use the `, function() {` style instead?

In a recent project-wide update, we converted all these arrow references in the various `spec.js` files to the other style to be more in-line with Mocha.  So we'd like to have new adapters maintain this consistency in their spec files.

See https://github.com/prebid/Prebid.js/issues/2987 for additional context.